openedx_caliper_tracking/tests/test_utils.py

```diff
@@ -0,0 +1,120 @@
+from smtplib import SMTPException
+
+import mock
+from django.conf import settings
+from django.test import TestCase, override_settings
+
+from openedx_caliper_tracking import utils
+from openedx_caliper_tracking.tests.factories import UserFactory
+
+
+class CaliperUtilsTestCase(TestCase):
+
+    def test_convert_date_time(self):
+        self.assertEquals('2018-10-16T14:23:24.785Z', utils.convert_datetime('2018-10-16T14:23:24.785148+00:00'))
+
+    def test_get_username_from_user_id(self):
+        UserFactory(username='dummy')
+        self.assertEquals('dummy', utils.get_username_from_user_id(1))
```

Comment:
Please use the id of the user created from the above statement.

Reply:
done